.github/workflows/issue-management-stale-action.yml

```diff
@@ -34,3 +34,38 @@ jobs:
             as needing author feedback and has not had any activity for 7 days.
             It will be closed automatically if there is no response from the author
             within 7 additional days from this comment.
+
+      - uses: actions/stale@3a9db7e6a41a89f618792c92c0e97cc736e1b13f # v10.0.0
+        with:
+          days-before-stale: 365
+          days-before-close: -1 # the stale tag workflow will take care of actually closing this
+          stale-issue-label: "needs author feedback"
```

Comment:
I think I'd use `stale` here

it's more that no one picked it up, less that we need the author to comment more on it